lancero_source.go

```diff
@@ -125,6 +127,16 @@ func (ls *LanceroSource) updateChanOrderMap() {
 	}
 }
 
+// ConfigureMixFraction provides a default implementation for all non-lancero sources that
+// don't need the mix
+func (ls *LanceroSource) ConfigureMixFraction(processorIndex int, mixFraction float64) error {
+	if processorIndex >= len(ls.MixFraction) || processorIndex < 0 {
```

Comment:
I understand the simplicity of forcing clients to call this once per TES, but I'd like to suggest supporting the mode that if `processorIndex < 0` you set all mix values to the same level (presumably you'd only use this mode to set all mix to 0, but whatever).